inventory/debian.cf

```diff
@@ -23,6 +23,14 @@ bundle common inventory_debian
       "mint_release" string  => "$(mint_info[RELEASE][1])" ;
       "mint_codename" string => "$(mint_info[CODENAME][1])" ;
 
+    any::
+      "proc_routes" data => data_readstringarrayidx("/proc/net/route",
+                                                    "#[^\n]*","\s+",20,2048);
```

Comment:
Lets guard the parsing of this file on it's presence:

`ifvarclass => fileexists("/proc/net/route")`

Actually I think that ifvarclass can't use fileexists directly, so you might have to do something like `ifvarclass => fileexists("/proc/net/route")`